app/src/main/java/org/wikipedia/feed/suggestededits/SuggestedEditsFeedClient.kt

```diff
@@ -17,19 +29,39 @@ class SuggestedEditsFeedClient(
     private var clientJob: Job? = null
 
     override fun request(context: Context, wiki: WikiSite, age: Int, cb: FeedClient.Callback) {
+        clientJob?.cancel()
         this.age = age
         this.cb = cb
 
         if (UserContribStats.isDisabled() || UserContribStats.maybePauseAndGetEndDate() != null) {
             cb.success(emptyList())
             return
         }
+        val cardTypes = JsonUtil.decodeFromString<MutableList<DescriptionEditActivity.Action>>(Prefs.suggestedEditsFeedCardTypes) ?: mutableListOf()
+        if (cardTypes.isEmpty()) {
+            clientJob = coroutineScope.launch(Dispatchers.IO + CoroutineExceptionHandler { _, throwable ->
```

Comment:
We don't need to explicitly specify `Dispatchers.IO`. Network calls will be placed on the `IO` dispatcher automatically.